browser.es.js

```diff
@@ -1,3 +1,8 @@
-const scratchblocks = require("./index")(window)
+import init from "./index.js"
+
+const scratchblocks = init(window)
```

Comment:
For some reason I didn't think this would work (exporting the result of running a function that we've imported), but of course it does--nice!